packages/app_ui/lib/src/theme/app_theme.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:app_ui/app_ui.dart';
+import 'package:flutter/material.dart';
+
+abstract class AppTheme {
+  static final ThemeData lightTheme = ThemeData(
+    fontFamily: 'Rubik',
+    colorScheme: ColorScheme.fromSwatch().copyWith(
+      primary: AppColors.primary,
+      secondary: AppColors.accent,
+      background: AppColors.black[50],
+      surface: AppColors.white,
+      brightness: Brightness.light,
+    ),
+    iconTheme: _iconTheme,
+    appBarTheme: _appBarTheme,
+    textTheme: _textTheme,
+  );
+
+  static const _iconTheme = IconThemeData(color: AppColors.primary);
+
+  static final _appBarTheme = AppBarTheme(
+    titleTextStyle: AppTypography.headline3.copyWith(
```

Comment:
The style of the app bar's title is not following the material typography? You need to customize it?